test/integration/gateway_webhook_test.go

```diff
@@ -49,24 +48,7 @@ func TestGatewayValidationWebhook(t *testing.T) {
 
 	waitForWebhookService(t)
 
-	t.Log("creating a gatewayclass to verify gateway validation")
-	gatewayc, err := gatewayclient.NewForConfig(env.Cluster().Config())
-	require.NoError(t, err)
-	gatewayClass := &gatewayv1alpha2.GatewayClass{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: uuid.NewString(),
-		},
-		Spec: gatewayv1alpha2.GatewayClassSpec{
-			ControllerName: gateway.ControllerName,
-		},
-	}
-	gatewayClass, err = gatewayc.GatewayV1alpha2().GatewayClasses().Create(ctx, gatewayClass, metav1.CreateOptions{})
-	require.NoError(t, err)
-
-	defer func() {
-		t.Logf("cleaning up gatewayclass %s", gatewayClass.Name)
-		assert.NoError(t, gatewayc.GatewayV1alpha2().GatewayClasses().Delete(ctx, gatewayClass.Name, metav1.DeleteOptions{}))
-	}()
+	gatewayClient, err := gatewayclient.NewForConfig(env.Cluster().Config())
```

Comment:
Why was error check require.NoError(t, err) removed?

```suggestion
	gatewayClient, err := gatewayclient.NewForConfig(env.Cluster().Config())
	require.NoError(t, err)
```

Reply:
Whoops, it should be checked, going to add it.